app/router.js

```diff
@@ -8,6 +8,7 @@ const Router = EmberRouter.extend({
 
 Router.map(function () {
   this.route('confirm-user', { path: '/confirm-user/:token' });
+  this.route('travis-ci-insights');
```

Comment:
remove. There are already routes t the end of this file

Reply:
At the end of this there routes for insights which was already present in the past also. we do have added new route travis-ci-insights and there is only one instance of this in this whole file

Reply:
@RaviRanjan86 why you marked this as resolved?
The page should be rendered under /insights URL, and its tab should be before "Notifications" tab. You should re-use this route:
`this.route('insights', { resetNamespace: true }, function () {`